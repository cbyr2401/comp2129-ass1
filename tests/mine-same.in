g 10 10
b 1 1
b 0 2
b 2 0
b 1 1
b 3 4
b 4 4
b 2 3