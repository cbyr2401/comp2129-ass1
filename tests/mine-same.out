g 10 10
b 1 1
b 0 2
b 2 0
error
